src/components/ImageSocialShare.vue

```diff
@@ -24,7 +24,7 @@ export default {
   }),
   computed: {
     imageURL() {
-      return this.image.foreign_landing_url;
+      return this.image.url;
```

Comment:
This `image.url` points to the .jpg of the image and the `foreign_landing_url` points to the image page on the source (which is what we need to share in this case). I think you need to find a different solution, maybe expose it via a different property

Reply:
Done @brenoferreira I added a new `prop` then